src/content/blog/docs-hacktoberfest.mdx

```diff
@@ -0,0 +1,45 @@
+
+Welcome to Astro Docs Week! All week, our docs maintainers are taking over [Astro’s Twitter account](https://twitter.com/astrodotbuild). We’ll be sharing docs updates and fun from the community, so make sure you’re following us!
+
+To kick things off, let’s take a moment to talk about Astro Docs and Hacktoberfest. We love to celebrate Astro’s amazing open-source contributors. Here’s how Hacktoberfest helped us do that.
+
+## Community-driven docs
+
+<BlogContentImage src={community} alt="Avatars for the many, many contributors to Astro docs" />
+
+[Astro’s documentation website](https://docs.astro.build/) is open source and driven by our community. Our [270+ contributors](https://github.com/withastro/docs/graphs/contributors) make it easier for you to use Astro every day and we couldn’t do it without them! So this year, we wanted to use Hacktoberfest as a chance to recognize the work they do.
+
+[Hacktoberfest](https://hacktoberfest.com/) is an annual event run by Digital Ocean to encourage open-source participation. Every October, people who sign up can see their PRs rewarded with a t-shirt — or have a tree planted in their name! For the first time, this year’s edition welcomed [low or non-code contributions](https://hacktoberfest.com/about/#low-or-non-code). How could we say no to a forest of docs contributions?
+
+## Planting the seed
+
+For us, contributions come in all shapes and sizes. From typo fixes and CSS tweaks, to bigger documentation improvements. But that’s not all! Astro’s Docs are available in 10 languages thanks to our tireless translators. That’s where we decided to focus our efforts this month.
+
+In line with Hacktoberfest’s low-code focus, we wanted to encourage new translations. But **we can’t merge translations without reviews from our multilingual community.** It might not be obvious, but a good PR review is as valuable as the PR itself. To recognize this, we made it possible to [get Hacktoberfest credit for reviewing PRs](https://github.com/withastro/docs/blob/main/.github/hacktoberfest.md). Then we let folks know our plans on [Discord](https://astro.build/chat) and [Twitter](https://twitter.com/astrodotbuild/status/1576221838950412289), sat back, and waited…
+
+## So, how did it go?
+
+Drum roll please... Astro’s docs merged **more than 130 PRs from 55 unique contributors** as part of this year’s Hacktoberfest! 🎉
+
+As we had hoped, more than half of these PRs were translation-related. Along the way we saw someone speed up our CI tests. Our awesome Spanish team matched the smallest changes to our English text word for word. (In fact, they’re likely translating this blog post as you read this!) Deploy guides for Deno Deploy and GitHub Pages got a once over. A new contributor submitted an epic, seven-page Russian translation. A bug in our docs Discord bot got squashed. Our community made improvements across all our languages. And did typos get fixed? You bet!
```

Comment:
```suggestion
As we had hoped, more than half of these PRs were translation-related. And, along the way:

- Someone sped up our CI tests. 
- Our awesome Spanish team battled against incoming PRs to maintain "translation inbox zero." (In fact, they’re likely translating this blog post as you read this!) 
- Deploy guides for Deno Deploy and GitHub Pages got a once over. 
- A new contributor submitted an epic, seven-page Russian translation. 
- A bug in our docs Discord bot got squashed.
- Our community made improvements across all our languages. 
- . . . And did typos get fixed? You bet!
```
Just wondering whether a list might be easier to scan/call out the achievements? (And, I didn't change much text, but this allows us to care less about how it all flows as one paragraph.)

Reply:
Yes, I wondered about this too — let’s try and see how it looks on the preview! (Removing the ellipsis here)

```suggestion
As we had hoped, more than half of these PRs were translation-related. And, along the way:

- Someone sped up our CI tests. 
- Our awesome Spanish team battled against incoming PRs to maintain "translation inbox zero." (In fact, they’re likely translating this blog post as you read this!) 
- Deploy guides for Deno Deploy and GitHub Pages got a once over. 
- A new contributor submitted an epic, seven-page Russian translation. 
- A bug in our docs Discord bot got squashed.
- Our community made improvements across all our languages. 
- And did typos get fixed? You bet!
```

The only question in my mind is if a list implies a more exhaustive presentation? There’s more we _could_ add here. Should we?